posthog/queries/session_recordings/session_recording_list_from_replay_summary.py

```diff
@@ -37,29 +37,36 @@ class SessionRecordingListFromReplaySummary(SessionRecordingList):
     )
     """
 
+    _session_recordings_base_query: str = """
+    SELECT
+       session_id,
+       any(team_id),
+       any(distinct_id),
+       min(min_first_timestamp) as start_time,
+       max(max_last_timestamp) as end_time,
+       dateDiff('SECOND', start_time, end_time) as duration,
+       argMinMerge(first_url) as first_url,
+       sum(click_count),
+       sum(keypress_count),
+       sum(mouse_activity_count),
+       round((sum(active_milliseconds)/1000)/duration, 2) as active_time
+    FROM session_replay_events
+    PREWHERE team_id = %(team_id)s
```

Comment:
Another small suggestion:

The `person_cte_clause` inside prewhere is borking things, since it forces a subquery for a prewhere clause

https://posthog.com/handbook/engineering/clickhouse/data-storage#no-query-planner

I'd recommend moving everything after `max_last_timestamp` to a WHERE clause instead, should further improve performance here.

Reply:
ah, session ids can stay, that's ideal.


(80% confidence in the `cte_clause` suggestion, without doing any benchmarking, so trust but verify if you'd like to 🙈 )

Reply:
👀 💖